compiler/rustc_pattern_analysis/Cargo.toml

```diff
@@ -1,7 +1,9 @@
+cargo-features = ["edition2024"]
+
 [package]
 name = "rustc_pattern_analysis"
 version = "0.0.0"
-edition = "2021"
+edition = "2024"
```

Comment:
I guess it may be worthwhile to revert the changes to pattern analysis if R-A needs it to build.

Reply:
I can try when I’m back from holidays but do you know if the rustc-crates-on-stable rmake test failed as we expect with the pattern analysis changes? I wonder if using some of cargo’s unstable would make it fail. If not, I’d like to fix it.